plugins/aggregators/quantile/quantile.go

```diff
@@ -62,7 +62,7 @@ func (q *Quantile) Add(in telegraf.Metric) {
 	}
 	for k, field := range in.Fields() {
 		if v, isconvertible := convert(field); isconvertible {
-			// This should never error out as we tested it in Init()
+			//nolint:errcheck // This should never error out as we tested it in Init()
```

Comment:
Maybe instead of the comment we bite the bullet and do the error check here, event if it can't fail... What do you think?

Reply:
Error check can involve `q.Log.Errorf(...)`